keyboards/kbdfans/odin75/info.json

```
@@ -0,0 +1,126 @@
+{
+    "manufacturer": "KBDFans",
+    "keyboard_name": "ODIN75",
+    "maintainer": "KBDFANS",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "oled": true,
+        "rgblight": false,
+        "wpm": true
+    },
+    "indicators": {
+        "caps_lock": "GP29",
+        "on_state": 1
+    },
+    "matrix_pins": {
+        "cols": ["GP19", "GP18", "GP17", "GP16", "GP15", "GP14", "GP13", "GP12", "GP11", "GP10", "GP9", "GP8", "GP7", "GP6", "GP3", "GP4"],
+        "rows": ["GP0", "GP1", "GP2", "GP21", "GP20", "GP5"]
+    },
+    "processor": "RP2040",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x6072",
+        "vid": "0x4B42"
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                { "matrix": [0, 0], "w": 1, "x": 0, "y": 0 },
```

```suggestion
                { "matrix": [0, 0], "x": 0, "y": 0 },
```
etc...
`w: 1` is the default.